R/getSymbols.R

```diff
@@ -599,7 +599,7 @@ function(Symbols,env,return.class='xts',
                                db.fields=c('date','o','h','l','c','v','a'),
                                field.names = NULL,
                                user=NULL,password=NULL,dbname=NULL,host='localhost',port=3306,
-                               ...) {
+                               from=NULL, to=NULL, ...) {
```

Comment:
These new arguments need to be documented in the corresponding .Rd file. You can use `man/getSymbols.yahoo.Rd` as a template. Though I'm not sure the defaults for `getSymbols.yahoo` make sense here.